flink-runtime/src/main/java/org/apache/flink/runtime/scheduler/adaptive/AdaptiveScheduler.java

```diff
@@ -784,7 +789,7 @@ public ArchivedExecutionGraph getArchivedExecutionGraph(
     }
 
     @Override
-    public void goToWaitingForResources() {
+    public void goToWaitingForResources(@Nullable ExecutionGraph executionGraph) {
```

Comment:
nit: `previousExecutionGraph`

Reply:
Done in aad45c9f66e0cad768be2f8172b569f61ace3477.